pages/collective-page.tsx

```diff
@@ -193,7 +193,7 @@ export default function CollectivePage(props: InferGetServerSidePropsType<typeof
                 onPrimaryColorChange={onPrimaryColorChange}
                 step={step}
                 mode={mode}
-                refetch={data.refetch}
+                refetch={queryResult.refetch}
```

Comment:
This is the fix for https://github.com/opencollective/opencollective/issues/7905